about/contributing_datasets.md

```diff
@@ -0,0 +1,288 @@
+---
+layout: default
+title: Contributing datasets
+nav_order: 1
+parent: About Data Commons
```

Comment:
Good idea to start a new section for this, though I'm not sure if it should be "about" or "contributing" (we can migrate the other docs to go there). What do you think?

Reply:
I might prefer having Contributing to Data Commons section (as a counterpart to API).

Reply:
Could you also migrate the other MD files from https://github.com/datacommonsorg/data/tree/master/docs in the same directory? A separate PR is fine, too.

Reply:
I like the idea of having a new section called 'Contributing,' parallel to the API section and linked at the bottom of the left menu; I will go ahead and migrate the other MD files too.

Reply:
https://github.com/datacommonsorg/docsite/pull/63